### ISSUE TITLE
Compatability with Storybook 6

### ISSUE THREAD

**Issue by houfio** [atanasster/storybook-addon-deps#8]
Since alpha 15 of Storybook 6 the plugin fails to initialize due to an error [on this line](https://github.com/atanasster/storybook-addon-deps/blob/555d3c132dab4615e1d080d8e01424778213a5b7/src/shared/utils.ts#L90).

This is the error:
```
Cannot read property 'parameters' of null
TypeError: Cannot read property 'parameters' of null
    at http://localhost:8000/vendors~main.ddcfb952246300a85a56.bundle.js:178543:55
    at Array.find (<anonymous>)
    at http://localhost:8000/vendors~main.ddcfb952246300a85a56.bundle.js:178542:39
    at Array.map (<anonymous>)
    at mapModuleToStory (http://localhost:8000/vendors~main.ddcfb952246300a85a56.bundle.js:178539:30)
    at getDependenciesProps (http://localhost:8000/vendors~main.ddcfb952246300a85a56.bundle.js:178598:15)
    at http://localhost:8000/vendors~main.ddcfb952246300a85a56.bundle.js:178111:107
    at updateContextConsumer (http://localhost:8000/vendors~main.ddcfb952246300a85a56.bundle.js:143422:19)
    at beginWork$1 (http://localhost:8000/vendors~main.ddcfb952246300a85a56.bundle.js:143805:14)
    at HTMLUnknownElement.callCallback (http://localhost:8000/vendors~main.ddcfb952246300a85a56.bundle.js:123914:14)
```

The devs of Storybook have written up a nice document explaining what's changed [here](https://github.com/storybookjs/storybook/compare/v6.0.0-alpha.14...v6.0.0-alpha.15#diff-177dcef0a5b481c2c87db87f7cdf200d).

(and what's up with v1.0.12? It's up on npm, but the repo is still on v1.0.11 ;))

**Comment by atanasster** [comment]
@houfio thanks for the report, 0.11 was just released yesterday and i now pushed to github.

As for the sb alphas, i haven't checked for now and cant confirm if it breaks my addon - probbaly when 6.0 enters into beta if it still breaks existing addons i will look for a solution.

**Comment by atanasster** [comment]
Ill close this for now - while storybook 6.0 is in alpha best to report to them any incompatibility issues, it might be inadvertently introduced.
If still an issue when 6.0 enters beta, please open a new issue.

**Comment by houfio** [comment]
Since they've put it in their [migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md) I'm sure the change is there to stay. I'll lock my Storybook versions for now and hope they get to beta soon. Thanks for all your work!

**Comment by atanasster** [comment]
Thanks for understanding. Yeah its bit meaningless to chase changes in alphas, got to give them the space to reverse decisions etc. and lets see what happens in the betas.

**Comment by puyt** [comment]
Storybook 6 is out of beta, any idea when it's gonna be supported?
Thanks in advance!

**Comment by atanasster** [comment]
Thanks for the heads up, i will give it a try over the week end.

At some point i did make the addon compatible with 6beta, have you given it a try with the 6.xx updates? It might work, but they also broke compatibility multiple times with 6 so i wouldn't be surprised if i need to adjust it again.

**Comment by puyt** [comment]
@atanasster Great!

I've tested it with the latest release and I get the following error because they've renamed `PropsTable` to `ArgsTable`, not sure if they're any other issues.

![image](https://user-images.githubusercontent.com/3668615/90230008-2dabbd80-de19-11ea-88d8-f9d06ad62bae.png)

**Comment by atanasster** [comment]
Great thx a lot - if you want to submit a pr or i will get to it over the week end

**Comment by atanasster** [comment]
Just released 2.1.0 dealing with sb6 later stage incompatibilities introduced during the rc period.

If you find any issues, please create a new issue.
